onnxscript/rewriter/ort_fusions/gelu.py

```diff
@@ -34,4 +34,5 @@ def rewrite(self, op, x):
 
 
 def fuse_gelu(model: ir.Model, debug: bool = False) -> int:
-    return _fusion_utils.apply_fusion_rules(gelu_rules, model, debug=debug)
+    fuse_gelu = _fusion_utils.apply_fusion_rules(gelu_rules)
```

Comment:
A nit clarification: what I meant was that these 3 lines can be replaced by `fuse_gelu = _fusion_utils.apply_fusion_rules(gelu_rules)` no need for lines 36 or 38. (You will still need some extra lines in cases where there is an extra `remove_unused_nodes` ... which will eventually disppear, because it is not strictly necessary, but that is a separate cleanup, don't want to overload this PR with other unrelated changes.)